src/web/mjs/connectors/SakuraNovel.mjs

```diff
@@ -0,0 +1,81 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+//template/theme LightNovel, basically WordPressMangastreamNovel
+export default class SakuraNovel extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'sakuranovel';
+        super.label = 'SakuraNovel';
+        this.tags = ['novel', 'indonesian'];
+        this.url = 'https://sakuranovel.id';
+
+        this.querMangaTitleFromURI = 'div.series-title h2';
+        this.queryMangas = 'div.novellist-blc ul li a.series';
+        this.queryChapters = 'ul.series-chapterlists li a';
+        this.queryChaptersTitle = 'span:not(.date)';
+        this.path = '/daftar-novel/';
+
+        this.novelContentQuery = 'div.asdasd';
```

Comment:
I have the feeling that this selector is not good

Reply:
![image](https://github.com/manga-download/hakuneko/assets/10671518/d3d67206-d49e-4312-a22b-1d25bdc0ede3)

Reply:
Please test your code before submitting, its not working.

Reply:
Well its better the thing is i wonder why i works on some novels and not on others, like i see in the screen.

🤔